tokio-console/src/config.rs

```diff
@@ -377,7 +377,7 @@ impl ViewOptions {
     /// - Checking the `terminfo` database via `tput`
     pub(crate) fn determine_palette(&self) -> Palette {
         // Did the user explicitly disable colors?
-        if self.no_colors.unwrap_or(true) {
```

Comment:
Alternatively, I think we could have just changed this to `unwrap_or(false)`...but I think your change is better. I don't love that the previous code had the CLI flag take a value, like `--no-colors true`, which seems quite weird. This is definitely better.